src/coreComponents/common/Units.hpp

```diff
@@ -33,12 +33,14 @@ namespace units
  * @return the input Kelvin degrees converted in Celsius
  * @param kelvin degrees input
  */
+GEOS_HOST_DEVICE
 inline constexpr double convertKToC( double kelvin )
 { return kelvin - constants::zeroDegreesCelsiusInKelvin; }
 /**
  * @return the input Celsius degrees converted in Kelvin
  * @param celsius degrees input
  */
+GEOS_HOST_DEVICE
```

Comment:
While you're on this, there are plenty of warnings like
```
warning #20013-D: calling a constexpr __host__ function("const T1 &  ::std::max<double> (const T1 &, const T1 &)") from a __host__ __device__ function("geos::constitutive::ElasticOrthotropicUpdates::getShearModulus const") is not allowed. The experimental flag '--expt-relaxed-constexpr' can be used to allow this.
warning #20013-D: calling a constexpr __host__ function("max") from a __host__ __device__ function("getShearModulus") is not allowed. The experimental flag '--expt-relaxed-constexpr' can be used to allow this.
warning #20013-D: calling a constexpr __host__ function("const T1 &  ::std::max<double> (const T1 &, const T1 &)") from a __host__ __device__ function("geos::constitutive::ElasticTransverseIsotropicUpdates::getShearModulus const") is not allowed. The experimental flag '--expt-relaxed-constexpr' can be used to allow this.
warning #20013-D: calling a constexpr __host__ function("max") from a __host__ __device__ function("getShearModulus") is not allowed. The experimental flag '--expt-relaxed-constexpr' can be used to allow this.
```
That you could fix easily by calling `LvArray::math::max` instead of `std::max`.
Can you give it a go. It should be trivial 🤞 😉

Reply:
I replaced a few of these.